radio/src/targets/horus/CMakeLists.txt

```diff
@@ -89,7 +89,12 @@ if (PCB STREQUAL X10)
     add_definitions(-DRADIO_FAMILY_T16)
     set(DEFAULT_INTERNAL_MODULE MULTIMODULE CACHE STRING "Default internal module")
     option(BLUETOOTH "Support for bluetooth module" OFF)
+    option(INTERNAL_GPS "Support for internal ublox GPS" ON)    
```

Comment:
How is this specific to the u-blox GPS? Otherwise, sure, merge it. Just be warned - conventional commit formatting expects a semi-colon after the commit type specifier for correct formatting... so `feat(T16,T18): GPS & USART6 support`, or perhaps something more descriptive to end users like `feat(T16,T18): Add support for internal GPS if Bluetooth not enabled`

Reply:
Thanks. Indeed nothing specific about uBlox, std. NMEA is expected. I copy pasted the text from TX16S, but will change it.